devito/core/gpu_openmp.py

```diff
@@ -1,20 +1,27 @@
 from functools import partial, singledispatch
 
 import cgen as c
+from sympy import Function
 import numpy as np
 
 from devito.core.operator import OperatorCore
 from devito.data import FULL
 from devito.exceptions import InvalidOperator
-from devito.ir.iet import Callable, ElementalFunction, MapExprStmts
+from devito.ir.equations import DummyEq
+from devito.ir.iet import (Block, Call, Callable, ElementalFunction, FindNodes,
+                           LocalExpression, List, MapExprStmts, Transformer)
```

Comment:
nitpicking: it wouldn't be bad to logically group the imported objects by "type" 

* Block, Call, Callable, ElementalFunction, and **List** are IET notes
* the others are Visitors

So I'd write them in this order:

```
Block, Call, Callable, ElementalFunction,  List, FindNodes, LocalExpression, MapExprStmts, Transformer
```